specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2022-10-01-preview/examples/GetClusterExample.json

```diff
@@ -32,7 +32,20 @@
           "totalNodeCount": 2,
           "agentVersion": "0.1.0",
           "privateLinkState": "Disabled",
-          "provisioningState": "Succeeded"
+          "provisioningState": "Succeeded",
+          "azureHybridBenefit": "NotApplicable",
+          "distribution": "AKS", 
+          "distributionVersion": "1.0",
+          "additionalProperties": [
+            {
+              "propertyName": "distributionVersion",
+              "propertyValue": "1.0"
```

Comment:
the field "propertyValue" is of type 'object'. Does it show up like a string in the response?